arch-template.json

```diff
@@ -22,6 +23,8 @@
                 "/usr/bin/curl -O http://{{ .HTTPIP }}:{{ .HTTPPort }}/poweroff.timer<enter><wait5>",
                 "/usr/bin/bash ./enable-ssh.sh<enter>"
             ],
+            "cpus": 1,
+            "memory": 768,
```

Comment:
\> 512 MB required to boot an install iso?

*\*sad old-timer noises\**

Reply:
512MB is at least the default in packer, but I couldn't boot the current install CD with 512MB. However, it was working in the past. Either it's related to VirtualBox or the ISO.

Reply:
I believe you. The archlinux install guide claims (oddly specific) 530 MiB RAM is required

> Arch Linux should run on any x86_64-compatible machine with a minimum of 530 MiB RAM.

https://wiki.archlinux.org/index.php/installation_guide#:~:text=Arch%20Linux%20should%20run%20on,working%20internet%20connection%20is%20available.